genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -135,9 +135,13 @@ to work.
 |false
 
 |genie.jobs.users.activeLimit.count
-|The maximum number of active jobs a user is allowed to have. Once a user hits this limit, jobs submitted are rejected. This is property is ignored unless `genie.jobs.users.activeLimit.enabled` is set to true.
+|The maximum number of active jobs a user is allowed to have. Once a user hits this limit, jobs submitted are rejected. This is property is ignored unless `genie.jobs.users.activeLimit.enabled` is set to true. This limit applies to users that don't have a more specific value set, see below.
 |100
 
+|genie.jobs.users.activeLimit.specialUser.<user-name>
```

Comment:
Just a note: This is fine in Boot 1.5 but in boot 2.+ this will break their new [relaxed bindings](https://github.com/spring-projects/spring-boot/wiki/Relaxed-Binding-2.0) rules. I think in the `master` branch I changed `activeLimit` to `active-limit` already but I don't exactly remember.

Also instead of `specialUser` maybe just `whitelist`?

Reply:
Indeed, i know porting this to `master` will be slightly different. But trying to get the simplest possible change into `3.x.x`.

Whitelist usually denotes "you're on the list, go ahead" -type of behavior.
This is not. We can use this to set lower-than-default or higher-than-default.

Reply:
makes sense. `userOverride` then?

Reply:
(on the other side, I'm not a big fan of `specialUser` either. Any other idea?